specs/MultiProfile.md

```diff
@@ -29,6 +29,21 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **CoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will work not immediately but after the next navigation, and it can only
+apply for current WebView2, which means if we start a new WebView2 using the same profile, all the
+settings are default values and cannot be set from the profile. By adding password-autosave and
+general-autofill management interfaces in profile, we can manage the properties and they will apply
```

Comment:
What does it mean by "will apply immediately"? Does it mean that if I set the property in the profile, it immediately applies the property to each associated WebView2, overriding the existing value of the property?

Reply:
Yes, the property in associated WebView2s will be overridden as well.